### PR TITLE
fix document in TFNet related examples

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

**Comment by yangw1234** [comment]
fixed #772 #767 #761

**Comment by yangw1234** [comment]
pr validation pass

File: pyzoo/zoo/examples/tensorflow/distributed_training/README.md [hunk 1]
```diff
@@ -18,7 +27,13 @@ export SPARK_HOME=... # Please edit this accordingly
...
+## Run the Evaluation Example after pip install
+
+```
+python evaluate_lenet.py
+```
+
+## Run the Evaluation Example with prebuilt package
 
 ```shell
 export ANALYTICS_ZOO_HOME=... # Please edit this accordingly
```
Comment (hkvision) [inline pyzoo/zoo/examples/tensorflow/distributed_training/README.md]: clarify this?

File: pyzoo/zoo/examples/tensorflow/tfnet/README.md [hunk 2]
```diff
@@ -13,29 +16,23 @@ In this example, we use `frozen_inference_graph.pb` of the `ssd_mobilenet_v1_coc
...
-    --master ${MASTER} \
-    --driver-memory 200g \
-    --executor-memory 200g \
-    --properties-file ${ANALYTICS_ZOO_CONF} \
-    --py-files ${ANALYTICS_ZOO_PY_ZIP},${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/tfnet/predict.py \
-    --jars ${ANALYTICS_ZOO_JAR} \
-    --conf spark.driver.extraClassPath=${ANALYTICS_ZOO_JAR} \
-    --conf spark.executor.extraClassPath=${ANALYTICS_ZOO_JAR} \
-    ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/tfnet/predict.py \
+sh $ANALYTICS_ZOO_HOME/bin/spark-submit-with-zoo.sh \
```
Comment (hkvision) [inline pyzoo/zoo/examples/tensorflow/tfnet/README.md]: what is ANALYTICS_ZOO_HOME?

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/tfnet/README.md [hunk 3]
```diff
@@ -20,15 +24,11 @@ master=... // spark master
...
 
-spark-submit \
---verbose \
+sh ${ANALYTICS_ZOO_HOME}/bin/spark-shell-with-zoo.sh \
 --master $master \
 --conf spark.executor.cores=1 \
 --total-executor-cores 4 \
---driver-memory 200g \
---executor-memory 200g \
 --class com.intel.analytics.zoo.examples.tfnet.Predict \
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/tfnet/README.md]: why remove --imagePath?

**Pull request merged**